### PR TITLE
fix/java version in CI

### PR DESCRIPTION
**Pull request by loulou2u**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Thanks for looking into the CI error @loulou2u!

I was under the impression the code should build under Java 11, even if it isn't quite fully compatible yet.
I'd ideally like to keep it a part of CI if it isn't the root cause.

It looks like the issue has been appearing in the Java 8 build for a while https://github.com/uPortal-Project/uPortal/actions/runs/3715033479/jobs/6299708747#step:3:4

Perhaps the fix for Java 8 will resolve the issue in Java 11 as well?

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ jobs:
           - zulu
         java-version:
           - 8
-          - 11
```
Comment (ChristianMurphy) [inline .github/workflows/CI.yml:24]: Do the tests still break on Java 11 with the dependency changes?
If not would it make sense to re-add it to the test matrix?

**Review by ChristianMurphy — APPROVED** [review]
Thanks @loulou2u!

File: .github/workflows/CI.yml (hunk 1 above)
Comment (loulou2u) [reply .github/workflows/CI.yml:24]: I'm re-adding Java 11 back to this file. I found that there were 2 dependency upgrades that were causing the tests to fail, despite it working after performing a dataInit, portalInit, and tomcatDeploy.

File: gradle.properties [hunk 2]
```diff
@@ -42,7 +42,7 @@ antVersion=1.10.12
 awsVersion=1.12.352
 apereoPortletUtilsVersion=1.1.3
 aspectjVersion=1.9.9.1
-casClientVersion=3.6.4
+casClientVersion=3.6.2
```
Comment (bjagg) [inline gradle.properties:45]: Thanks for tracking down these issues!

**Review by bjagg — APPROVED** [review]
🎖️

**Review by ChristianMurphy — APPROVED** [review]
:shipit:

**Pull request merged**